MICS6_Master.do

```diff
@@ -120,7 +120,8 @@ foreach name in $newMICS6countries {
 
 * Housekeeping
 	drop country_name
-	
+			pause on
+		pause RW itn
```

Comment:
Could you please clarify why we need the pause on command here?

Reply:
Corrected. please see new merge.